identity-iota/src/document/iota_document_metadata.rs

```diff
@@ -29,9 +28,9 @@ pub struct IotaDocumentMetadata {
     skip_serializing_if = "MessageIdExt::is_null"
   )]
   pub previous_message_id: MessageId,
-  /// `VerifiableProperties` contains the proof / signature section.
-  #[serde(flatten)]
-  pub properties: VerifiableProperties,
+  #[serde(skip_serializing_if = "Option::is_none")]
+  pub proof: Option<Signature>,
+  pub properties: Object,
```

Comment:
Seems like `VerifiableProperties` can be removed then. Only usage left is in a test.

Reply:
I tried to remove it but it's used as a proxy in the Wasm bindings for signing and verifying arbitrary data. We could move it to the Wasm bindings then like I did with `serde_into` but it could be useful for other bindings in the future too? Not sure.

Reply:
Did not expect that it was used there. Yeah, why not keep it for future bindings too, then.